crates/rspack_error/tests/out_of_order/multi_json_syntax_error/fixtures__multi_json_syntax_error.snap

```diff
@@ -2,15 +2,16 @@
 source: crates/rspack_error/tests/fixtures.rs
 expression: multi_json_syntax_error
 ---
-  × Error[json]: Json parsing error
-   ╭─[tests/out_of_order/multi_json_syntax_error/a.json:2:1]
```

Comment:
can we keep the path, maybe in future PR?

Reply:
Acutlly, we can and official swc-loader keeps the path too. But in both diagnostics of terminal and overlay, paths are formatted using `moduleName` and `moduleIdentifier` of `StatsError/Warning`. So it might get duplicated.

Reply:
okay